spec/date.html

```diff
@@ -558,6 +551,18 @@ <h1>PadYear ( _y_ )</h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-temporal-datetostring" aoid="DateToString">
+      <h1>DateToString ( _date_ )</h1>
+      <emu-alg>
+        1. Assert: Type(_date_) is Object.
+        1. Assert: _date_ has an [[InitializedTemporalDate]] internal slot.
+        1. Let _year_ be ! PadYear(_date_.[[Year]]).
```

Comment:
As a side note, I don't think the definition for PadYear is currently correct, but that doesn't have any bearing on whether or not to merge this.